docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -34,72 +47,58 @@ To build an APB:
 make apb_build
 ....
 
-Every APB has a Docker Hub repository set up for hosting images.
-When a PR for an APB repository is merged to master, the Docker Hub respository detects this and kicks off an automated build.
-If the build is successful, the resulting image is tagged as `latest`, replacing the previous `latest` image for that APB.
+Alternatively, if you have set up the alias described above you can directly use `apb build`.
 
-=== Setting up an Automated Build for APB Images
+=== Push an APB to a Local Openshift Cluster
 
-IMPORTANT: Docker repositories for APBs need to be created as an 'Automated Build' rather than a 'Repository'. There doesn't seem to be a way to add an automated build to a repository afterwards.
+Assuming your OpenShift cluster is up and running, you can push the APB image to the local OpenShift Docker Registry with:
 
-While signed into Docker Hub from a browser:
+....
+apb push -o
+....
 
-* Create > Create Automated Build
-* Choose Github and allow access to repositories in the aerogearcatalog Github org
-* Look for the repo in the list and click it
-* Make sure the Repository Namespace & Name are correct e.g. aerogearcatalog/metrics-apb. Visibility public is fine.
-* After creating, go to Build Settings
-* Configure 'master' branch to build
-* Configure tags that match a name of `/^[0-9.]+/` to build
-* Save changes and Trigger a build for `master`. If successful, there should be an image with the 'latest' tag.
+Afterwards your APB is ready to be used from the _Service Catalog_.
 
-=== Pushing to Dockerhub
+[NOTE]
+====
+* When pushing an APB it will not be available immediately. 
```

Comment:
Maybe a note about how long it might take, and why that is?

Reply:
Is that the case? The apbs are automatically relisted after a push and when i refresh the Service Catalog it's there. The only thing that can take a while is when you update details in apb.yml (description, icon, etc.)

Reply:
How long isn't deterministic. around 10-30 seconds. It's due to a bug: https://bugzilla.redhat.com/show_bug.cgi?id=1501523